### ISSUE TITLE
recall and tensorboard issue

### ISSUE THREAD

**Issue by muye5** [experiencor/keras-yolo3#43]
1. the latest training code, tensorboard doesn't show validation loss, only training loss_[1-3] left?
2. when I trained with my own dataset which contained only one class(person), I got some wrong boxes which contained other things like dog or horse with label person. I think the model detected some objects but cannot figure out they are not what i want. this derive a high recall but low precision. what can i do with this except level-up the threshold? Does this have a relation with the classification loss changing from softmax to cross-entropy?
